src/connect_dialog_handler.rs

```diff
@@ -241,39 +237,67 @@ impl ConnectDialog {
                 .padding(10)
                 .style(TEXT_BOX_CONTAINER_STYLE.get_container_style());
 
-        container(
-            column![column![
-                text("Connect To Remote Pi").size(20),
-                column![
-                    text_container,
-                    text(self.iroh_connection_error.clone())
-                        .style(CONNECTION_ERROR_DISPLAY.get_text_color()),
-                    text("Node Id").size(12),
-                    text_input("Enter node id", &self.node_id)
-                        .on_input(|input| Message::ConnectDialog(
-                            ConnectDialogMessage::NodeIdEntered(input)
-                        ))
-                        .padding(5),
+        let dialog_container = if self.disable_widgets {
+            container(
+                column![column![
+                    text("Connect To Remote Pi").size(20),
+                    column![
+                        text_container,
+                        text(self.iroh_connection_error.clone())
+                            .style(CONNECTION_ERROR_DISPLAY.get_text_color()),
+                        text("Node Id").size(12),
+                        text_input("Enter node id", &self.node_id).padding(5),
+                    ]
```

Comment:
These are formatting changes after I ran cargo fit, no?

Can you run "cargo fmt" on this branch and see if it changes them back?

Can you check you don't have any other formatter or formatter settings in your editor?

You don't have any custom cargo fmt settings or something else that would cause this reformatting?